### ISSUE TITLE
This project rocks !!! If there is time add an Influx DB (other DB) logging - this would make it a perfect tool :) !!!!!

### ISSUE THREAD

**Issue by lareooreal** [thomasnordquist/MQTT-Explorer#610]
I am really very happy how much time this tiny tool saved. 
Most Iot solutions are a pain in setting them up or debugging
the message que. 

Again - thanks for the excellent work !!!!!!!!!!!!!!!!!

**Comment by thomasvnl** [comment]
Just use NodeRED for that integration. MQTT-Explorer is meant to explore MQTT topics, not to log/store your data.

**Comment by tablatronix** [comment]
Yeah not sure the request here, you can use telegraf to instantly log to influx
